### ISSUE TITLE
Build and release multi-arch images to support arm64 and amd64 nodes

### ISSUE THREAD

**Issue by raelga** [openshift/external-dns-operator#231]
Currently, ExternalDNS Operator images are build and release for amd64 achitecure:

https://catalog.redhat.com/software/containers/edo/external-dns-rhel8/61d4c35023156829b87a434a?q=externaldns&architecture=amd64&image=65aab5615ad7bd5dc618b94a&container-tabs=overview
https://catalog.redhat.com/software/containers/edo/external-dns-operator-bundle/61d4c35112abd998d2073c4a?q=externaldns&architecture=amd64&image=65ae3661bba19a42335ee6c3
https://catalog.redhat.com/software/containers/edo/external-dns-rhel8-operator/61d4c35012abd998d2073c49?gs&q=externaldns

Adding support for arm64 images, will allow to deploy the operator in ARM64 nodes.

**Comment by alebedev87** [comment]
@raelga : right, we never had a requirement to support architectures other than amd64. Can you please file an RFE?

**Comment by raelga** [comment]
RFE created: https://issues.redhat.com/browse/RFE-6561

**Comment by alebedev87** [comment]
The RFE takes over from here, closing the issue.
